bin/build-coal-image

```diff
@@ -195,11 +199,17 @@ function create_output
     fi
 
     echo "==> Unpacking VMWare image... "
-    rm -rf $STAGE/USB-headnode.vmwarevm
-    (cd $STAGE && ${TAR} -jxvf ${vmwarevm_tarball}) \
+    rm -rf "$STAGE/USB-headnode.vmwarevm"
+    (cd "$STAGE" && ${TAR} -jxvf "${vmwarevm_tarball}") \
         || fatal "Unable to unpack image"
     echo "done."
 
+    echo -n "==> Setting Guest OS to \"Other 64-bit\"..."
+    sed -e 's/^guestOS.*/guestOS = "other-64"/' \
```

Comment:
The sed you have now will only affect the first time it encounters `guestOS`.  If that's what you intended, cool.  If you worry about multiple lines like this, perhaps add `g` after the final slash?